src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -1061,14 +1113,23 @@ private JTable getJTableInventoryRow() throws OHServiceException {
 				@Override
 				public void valueChanged(ListSelectionEvent e) {
 					if (e.getValueIsAdjusting()) {
-						jTableInventoryRow.editCellAt(jTableInventoryRow.getSelectedRow(), jTableInventoryRow.getSelectedColumn());
-						int[] selectedRows = jTableInventoryRow.getSelectedRows();
-						if (selectedRows.length == 1) {
-							MedicalInventoryRow medInvRow = (MedicalInventoryRow) jTableInventoryRow.getValueAt(selectedRows[0], -1);
-							if (medInvRow.getLot() == null || medInvRow.isNewLot()) {
+						int selectedRow = jTableInventoryRow.getSelectedRow();
+						if (selectedRow != -1) {
+							MedicalInventoryRow medInvRow = (MedicalInventoryRow) jTableInventoryRow.getValueAt(selectedRow, -1);
+							Lot lot = medInvRow.getLot() != null ? medInvRow.getLot() : null;
```

Comment:
Shouldn't be equal to this?

```suggestion
							Lot lot = medInvRow.getLot();
```